### DISCUSSION TITLE
I think nextJS example middleware authorizes  you if you use an expired jwt

### DISCUSSION THREAD

**Discussion by yoty66** [nextauthjs/next-auth#7384] [Help] [answered]
### Question 💬

It's just me or the middleware in the nextJS example allows you to pass an expired jwt?

[link](https://github.com/nextauthjs/next-auth/blob/ddffa57d00664bc6ba7990a9b6c3bfe2f131ebe6/apps/examples/nextjs/middleware.ts)

```
import { withAuth } from "next-auth/middleware"

// More on how NextAuth.js middleware works: https://next-auth.js.org/configuration/nextjs#middleware
export default withAuth({
  callbacks: {
    authorized({ req, token }) {
      // `/admin` requires admin role
      if (req.nextUrl.pathname === "/admin") {
        return token?.userRole === "admin"
      }
      // `/me` only requires the user to be logged in
      return !!token
    },
  },
})

export const config = { matcher: ["/admin", "/me"] }
```

Shouldn't it be changed to ? 

```
      // `/me` only requires the user to be logged in
      return !!token &&  token?.exp as number >new Date().getTime()
```

### How to reproduce ☕️

1.Run the nextJs example.
2.sign in. 
3. copy the jwt and wait for it to expire 
4. go to the /me page 

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by balazsorban44** [comment, marked as answer]
`token.exp` is not a default value. `token.expires` is, but when that would evaluate to true, the session token is actually cleared from cookies.
